packages/calypso-e2e/src/media-helper.ts

```diff
@@ -78,39 +79,37 @@ export function createTestFile( {
 	const sourceFileDir = path.join( __dirname, '../../../../../test/e2e/image-uploads/' );
```

Comment:
This is outside the scope of this PR, but we should really refactor this out. A package shouldn't need to have knowledge of the directory structure of the project using said package. Instead, this path should come as part of `sourceFileName`

Reply:
With the EPERM issues resolved, another solution (that was originally intended) could be to have an `image-source` directory in this package which holds the test files.

How does that sound?